paper-server/src/main/java/org/bukkit/craftbukkit/CraftLootTable.java

```diff
@@ -86,32 +87,29 @@ public Collection<ItemStack> populateLoot(Random random, LootContext context) {
     public void fillInventory(Inventory inventory, Random random, LootContext context) {
         Preconditions.checkArgument(inventory != null, "Inventory cannot be null");
         Preconditions.checkArgument(context != null, "LootContext cannot be null");
-        LootParams nmsContext = this.convertContext(context, random);
+        LootParams nmsContext = this.convertContext(context);
         CraftInventory craftInventory = (CraftInventory) inventory;
         Container handle = craftInventory.getInventory();
 
         // TODO: When events are added, call event here w/ custom reason?
-        this.getHandle().fillInventory(handle, nmsContext, random.nextLong(), true);
+        this.getHandle().fill(handle, nmsContext, new RandomSourceWrapper(random), true);
```

Comment:
random is Nullable in the API, so you get a NPE here and above

Reply:
This was broken before, wasn't it? I pushed a fix lol